pygeoapi/provider/base.py

```diff
@@ -313,3 +313,8 @@ class ProviderVersionError(ProviderGenericError):
 class ProviderInvalidDataError(ProviderGenericError):
     """provider invalid data error"""
     pass
+
+
+class ProviderRequestEntityTooLarge(ProviderGenericError):
```

Comment:
Rename to `ProviderRequestEntityTooLargeError`

Reply:
Ups, missed that part, sorry!